net-dns/knot/metadata.xml

```diff
@@ -10,9 +10,18 @@
 		<name>Proxy Maintainers</name>
 	</maintainer>
 	<use>
+		<flag name="authsignal">
+			Enable Automatic Authenticated DNSSEC Bootstrapping records
+		</flag>
 		<flag name="cookies">
 			Enable the DNS cookie module (RFC 7873)
 		</flag>
+		<flag name="daemon">
+			Enable Knot DNS main daemon
+		</flag>
+		<flag name="dbus">
+			Enable DBus support as an alternative of systemd d-bus
```

Comment:
I'm not sure if I follow this. Also, dbus and pkcs11 are global USE flags. You can provide a local definition here if it's useful but these two definitions don't seem to add much information either...

Reply:
@thesamesam, it seems pkcs11 is not global : [use.decs](https://github.com/gentoo/gentoo/blob/master/profiles/use.desc#L262).

I deleted dbus.